tests/test_utils.py

```diff
@@ -27,3 +28,64 @@ class Emitter(QObject):
         qtbot.wait(10)
 
     receiver.assert_not_called()
+
+
+def test_get_max_args_simple():
+    def fun1():
+        pass
+
+    assert get_max_args(fun1) == 0
+
+    def fun2(a):
+        pass
+
+    assert get_max_args(fun2) == 1
+
+    def fun3(a, b=1):
+        pass
+
+    assert get_max_args(fun3) == 2
+
+    def fun4(a, *, b=2):
+        pass
+
+    assert get_max_args(fun4) == 1
+
+    def fun5(a, *b):
+        pass
+
+    assert get_max_args(fun5) is None
```

Comment:
```suggestion
    assert get_max_args(fun5) is None
    
    assert get_max_args(print) is None
```

as we know that print is valid test case.

Maybe also add test for raising on int?